guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -188,12 +191,12 @@ uid = BaseOS
 +
 . Open the `/var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart/treeinfo` file for editing.
 . In the `[general]` section, make the following changes:
-* Change `packagedir = AppStream/Packages` to `packagedir = Packages`
+* Change `packagedir = AppStream/Packages` to `packagedir = Packages`.
 * Change `repository = AppStream` to `repository = .`
```

Comment:
On those lines dots at the end of the sentences are confusing. It is fine to not have dots at the end of this is consistent throughout all bullet in series. Please revert this change.

Reply:
It was not consistent in the first place. I have now removed all dots in the list of bullet points by effectively changing one line.
Let me know if you're happy with this fixup commit and I can squash it before merging.